### DISCUSSION TITLE
Would parliament be acceptable?

### DISCUSSION THREAD

**Discussion by BastianZim** [Homebrew/discussions#613] [Everyday usage] [answered]
Hi,
I just wanted to quickly ask if the maintainers would accept a formula for [parliament](https://github.com/duo-labs/parliament)?

Reason for me asking is that it can be used as an [application](https://github.com/duo-labs/parliament#usage) and as a [library](https://github.com/duo-labs/parliament#using-parliament-as-a-library), the latter not being acceptable to homebrew (https://docs.brew.sh/Python-for-Formula-Authors).

Just wanted to check, before I start creating one.

Thanks!

**Comment by dtrodrigues** [comment, marked as answer]
At a glance it looks like a suitable candidate for homebrew-core. A PR would be welcome.

**Reply by BastianZim** [reply]
Perfect, thanks for the feedback!
